### PR TITLE
Add support to configure custom seeder base class

### LINKED ISSUE #773: Configuration for base class used in SeedCreate command
It would be great to have a configuration option [`seed_base_class`](https://github.com/robmorgan/phinx/blob/df8933786806848fbf330f88d69998f29c29d8c4/src/Phinx/Console/Command/SeedCreate.php#L117-L121) just like [`migration_base_class`](https://github.com/robmorgan/phinx/blob/df8933786806848fbf330f88d69998f29c29d8c4/src/Phinx/Console/Command/Create.php#L210-L214).

### PR DESCRIPTION
**Pull request by glensc**
Replaces #898, Fixes #773

### CODE REVIEW

**Comment by glensc** [comment]
i wonder, if multiple authors modify same commit, should one be in author and other on signed off line?

in this case i kept original author, but added me to signed off line, should it be the opposite?

**Comment by dead23angel** [comment]
@glensc it was necessary to replace the author.

**Comment by glensc** [comment]
@dead23angel I do not understand what are you saying. I did not replace the author yet. Are you saying I should replace commit author with mine?

**Comment by garas** [comment]
GitHub annotated commit as "dead23angel _authored_ and glensc _committed_"

The _author_ is the person who originally wrote the patch, whereas the _committer_ is the person who last applied the patch. 

So current commit author and committer are correct.

**Comment by glensc** [comment]
@garas thanks for clarifying!

File: src/Phinx/Config/Config.php [hunk 1]
```diff
@@ -285,6 +285,19 @@ public function getSeedPaths()
         return $this->values['paths']['seeds'];
     }
 
+    /**
+     * Gets the base class name for seeders.
+     *
+     * @param boolean $dropNamespace Return the base sedder class name without the namespace.
```
Comment (dereuromark) [inline src/Phinx/Config/Config.php]: sedder => seeder

**Comment by dereuromark** [comment]
Looks good apart from my comment.

**Comment by dereuromark** [comment]
ping @glensc

**Comment by dereuromark** [comment]
@MasterOdin What do you think?

**Comment by MasterOdin** [comment]
@dereuromark it's essentially a straight copy of how the custom migration base class feature works, so cannot see why not.

**Comment by glensc** [comment]
@dereuromark ping what? it looked good already in august:
- https://github.com/cakephp/phinx/pull/1512#issuecomment-525682403

image

**Comment by garas** [comment]
Since August `docs/configuration.rst` was moved to `docs/en/configuration.rst` and it is now in conflict state.

Branches must be merged and the conflict resolved.

**Comment by glensc** [comment]
@MasterOdin rebased against `origin/master`

**Pull request merged**